torch/_inductor/ir.py

```diff
@@ -404,8 +404,19 @@ def get_reads(self):
                 ).reads
 
 
+def nop_loader_fn(idx, *, dtype):
+    if dtype in {torch.float16, torch.float32, torch.float64}:
```

Comment:
nit: maybe `if dtype.is_floating_point` (I think that exists)? unless bfloat16 should be excluded